crates/kitsune_p2p/kitsune_p2p/src/config.rs

```diff
@@ -46,16 +47,23 @@ pub(crate) struct KitsuneP2pTx2Config {
 pub struct KitsuneP2pConfig {
     /// List of sub-transports to be included in this pool
     pub transport_pool: Vec<TransportConfig>,
+
     /// The service used for peers to discover each before they are peers.
     pub bootstrap_service: Option<Url2>,
+
     /// Network tuning parameters. These are managed loosely,
     /// as they are subject to change. If you specify a tuning parameter
     /// that no longer exists, or a value that does not parse,
     /// a warning will be printed in the tracing log.
     #[serde(default)]
     pub tuning_params: KitsuneP2pTuningParams,
+
     /// The network used for connecting to other peers
     pub network_type: NetworkType,
+
+    /// Config for arc storage
+    #[serde(default)]
+    pub storage: LocalStorageConfig,
```

Comment:
I *think* you're setting / using the clamping from the tuning params, not from this config one level above the tuning params. Was this an earlier attempt that should be deleted?